.rubocop.yml

```diff
@@ -13,9 +10,18 @@ Metrics/BlockLength:
 Metrics/ClassLength:
   Enabled: false
 
+Metrics/CyclomaticComplexity:
```

Comment:
This codebase has its own RuboCop configuration instead of using the standard Panorama one. Let me know if you think I should switch it.